wolfssl/wolfcrypt/wc_port.h

```diff
@@ -1180,6 +1180,29 @@ WOLFSSL_ABI WOLFSSL_API int wolfCrypt_Cleanup(void);
     #endif
 #endif
 
+#ifdef WOLF_C99
+    /* use alternate keyword for compatibility with -std=c99 */
+    #define XASM_VOLATILE(a) __asm__ volatile(a)
+#else
+    #define XASM_VOLATILE(a) asm volatile(a)
+#endif
+
+#ifndef WOLFSSL_NO_FENCE
```

Comment:
Should we consider the fence off by default and on with `--enable-harden`?

Reply:
I don't see the downside of having it on by default. I don't think its a big performance consideration. Using `time` on the example client and server doesn't show any major differences. I think this might be a good on by default option.